MapboxCoreNavigation/CLLocation.swift

```diff
@@ -156,4 +156,14 @@ extension CLLocation {
         }
         return true
     }
+    
+    // Speed representation in miles per hour.
+    public var speedMPH: Double {
```

Comment:
I’d recommend spelling out “miles per hour”, but really this shouldn’t be a property of CLLocation. Instead, extend `CLLocationSpeed` with `milesPerHour` and `kilometersPerHour` properties, just as we extend `CLLocationDistance` with `miles` and `kilometers` properties.